chaincode/compute_plan_dag.go

```diff
@@ -90,7 +90,7 @@ func (dag *ComputeDAG) sort() error {
 			for _, c := range current {
 				errorIDs = append(errorIDs, c.ID)
 			}
-			return fmt.Errorf("compute plan error: Cyclic or missing dependency among inModels IDs: %v", errorIDs)
+			return errors.Internal("compute plan error: Cyclic or missing dependency among inModels IDs: %v", errorIDs)
```

Comment:
Could be a BadRequest as well: we don't support cyclic graph.

Reply:
Done